src/components/ItaliaTheme/View/Commons/PageHeader/PageHeader.jsx

```diff
@@ -59,6 +67,15 @@ const PageHeader = (props) => {
           </h1>
 
           <PageHeaderEventDates content={props.content} />
+          {props.content.stato_servizio !== null && (
+            <div class="chip chip-simple chip-primary mb-3">
+              <span class="chip-label">
+                {props.content.stato_servizio
+                  ? intl.formatMessage(messages.service_on)
+                  : intl.formatMessage(messages.service_off)}
+              </span>
+            </div>
```

Comment:
Hai provato ad usare direttamente il componente `Chip` di `design-react-kit`? Se funziona, preferirei usare quello.
Attenzione anche all'uso di `class` che in react non va bene, serve `className`.
https://italia.github.io/design-react-kit/?path=/docs/componenti-chips--varianti-di-colore